main.go

```diff
@@ -25,6 +30,30 @@ var (
 )
 
 func main() {
+	kingpin.MustParse(app.Parse(os.Args[1:]))
+	if bool(*force) == false {
+		files, err := ioutil.ReadDir(".")
+		if err != nil {
+			log.Fatal(err)
+		}
+		reader := bufio.NewReader(os.Stdin)
+		overwrite := false
+		for _, f := range files {
+			if f.Name() == "LICENSE" {
+				fmt.Print("There is already a license present in current directory. Do you want to overwrite it with a new one? [y/N] ")
```

Comment:
Right now, if there is no `LICENSE` at all and you run say `license-up mit Nikita Voloboev nikitavoloboev.xyz`. No license will get created.